lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -67,6 +67,7 @@ def self.eval_alert_expression(target, inputs, options = {})
   def self.build_evm_event(event, passed_inputs = {})
     inputs = passed_inputs.dup
 
+    # To Do: Add to Request Logs
```

Comment:
Use keywords (they show up in IDEs like VSCode and Sublime)

```suggestion
    # TODO: Add to Request Logs
```